src/test/ui/issues/issue-63151.rs

```diff
@@ -0,0 +1,19 @@
+// check-pass
```

Comment:
Can you please move this file to `src/test/ui/type-alias-enum-variants`, give the file name a more descriptive name `issue-63151-$short_description.rs` and also leave a comment in the file?

Reply:
@Centril, sure, done.

Reply:
I see these rules are described in the guidelines. Sorry, I should have looked there earlier. :) I just didn't recall there being any last time I contributed here around five years ago, that's why I didn't. ;-)

Reply:
No problem; thanks for the change and the PR!